### ISSUE TITLE
[Feature Request] Add images to "organization" page

### ISSUE THREAD

**Issue by ttaylor1on** [digitalocean/netbox#2934]
### Environment
* Python version:  
* NetBox version:

**Comment by ttaylor1on** [comment]
sorry I am new to GitHub. Thank you so much for your hard work. I would like to be able to add a Visio diagram to the "organization" page that is a very high level overview. If there is a existing way to do this sorry about that. I tied to look if this had been requested before but I did not see one. 

Thank you again

**Comment by DanSheps** [comment]
This issue has been closed as it does not conform to one of the [provided templates](https://github.com/digitalocean/netbox/issues/new/choose) as required by the [contributing guide](https://github.com/digitalocean/netbox/blob/master/CONTRIBUTING.md). If you'd like to request that your issue be re-opened, please first update the content so that it matches the appropriate template (this may require rewriting your issue entirely).

**Comment by ttaylor1on** [comment]
sorry I did not see the contributing guide. I am reading it now. That provided templates link does not take me a templates page.  Was the request invalid since I did not fill out form?

Environment
Python version: 
NetBox version: 
Proposed Functionality
Use Case
Database Changes
External Dependencies



those parts above this question?

I can fill those out now. I did not see where I could edit the original post until now.
